### PR TITLE
Sort correctly room collection.

### PR DESCRIPTION
**Pull request by Ivansss**
Rooms were not correctly sorted after multi-session PR https://github.com/nextcloud/spreed/pull/748.

### CODE REVIEW

**Comment by nickvergessen** [comment]
Still active room first, then by ping?

**Comment by Ivansss** [comment]
yes

File: js/models/roomcollection.js [hunk 1]
```diff
@@ -28,8 +28,15 @@
...
-			return [model.get('active') ? -1 : 0, -(model.get('lastPing'))];
+		comparator: function(modelA, modelB) {
+			var activeA = modelA.get('active'),
+			    activeB = modelB.get('active');
+
+			if (activeA !== activeB) {
+				return activeB - activeA;
+			}
+
+			return modelA.get('lastPing') < modelB.get('lastPing');;
```
Comment (nickvergessen) [inline js/models/roomcollection.js]: Remove double `;`

And also do math here? `modelB.get('lastPing') - modelA.get('lastPing')`

Comment (Ivansss) [reply js/models/roomcollection.js]: Done

**Pull request merged**